views/financial/income.pug

```diff
@@ -1,15 +1,69 @@
 extends ../base
+include ../_includes/yesNoRadios
 
 block variables
   -var title = 'Confirm your income information'
+  -var income = hasData(data, 'financial.income') ? data.financial.income : {}
+  -var taxes = hasData(data, 'financial.taxes') ? data.financial.taxes : {}
+  -var totalIncome = hasData(data, 'financial.totalIncome') ? data.financial.totalIncome : 0
+  -var totalTax = hasData(data, 'financial.totalTax') ? data.financial.totalTax : 0
+  - var count = 1;
+  - var year = new Date().getFullYear()-1
```

Comment:
You could hardcode this stuff if it were easier, but 🤷‍♂, also not bothered now it's done. 

Although would be better to keep formatting consistent.  (ie, `-var` vs `- var`)